code/control/DeployPlanDispatcher.php

```diff
@@ -178,6 +179,76 @@ protected function createFetch() {
 	}
 
 	/**
+	 * @param SS_HTTPRequest $request
+	 *
+	 * @return string
+	 */
+	public function deploysummary(SS_HTTPRequest $request) {
+
+		// Performs canView permission check by limiting visible projects
+		$project = $this->getCurrentProject();
+		if(!$project) {
+			return $this->project404Response();
+		}
+
+		// Performs canView permission check by limiting visible projects
+		$environment = $this->getCurrentEnvironment($project);
+		if(!$environment) {
+			return $this->environment404Response();
+		}
+
+		if(!trim($request->getBody())) {
+			$output = [
```

Comment:
Pretty minor, but you could save a few lines of code by just passing the output array straight into getAPIResponse rather than make a variable out of it, you don't use it anywhere else :)